src/main/java/Duke.java

```diff
@@ -122,6 +146,11 @@ public static void displayQuit(){
 
     //Add new task
     public static void addTaskIn(Task task,ArrayList<Task> tasks) {
+
+        if(task == null ){
```

Comment:
StringIndexOutOfBoundsException() seems to be forced here even though it has never occurred. Maybe throwing another type of exception e.g. MissingTaskException() will be more appropriate